Source/Terminals.Common/Connections/KnownConnectionConstants.cs

```diff
@@ -6,6 +6,8 @@ public class KnownConnectionConstants
 
         public const int HTTPPort = 80;
 
+        public const int HTTPSPort = 443;

```

Comment:
There should be no reason why to move it here, until there is another usage than tests. Leave it in https plugin.